recipes/mellon/meta.yaml

```
@@ -0,0 +1,44 @@
+{% set name = "mellon" %}
+{% set version = "1.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: b36b45ec9034c868c15210d9e497b3b6b55bdda4ee418dd1bfea0301b5fe0ef8
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - jax
+    - jaxopt
+    - scikit-learn
+
+test:
+  imports:
+    - mellon
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/settylab/mellon
+  summary: Non-parametric density estimator.
+  license: GPL-2.0-or-later
```

Looks like it's actually:
```suggestion
  license: GPL-3.0-or-later
```